.github/workflows/docs-deploy.yml

```
@@ -0,0 +1,41 @@
+      contents: write # for JamesIves/github-pages-deploy-action to push changes in repo
+    if: github.event_name != 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout️ 🛎
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
+          fetch-depth: 20
+          fetch-tags: false
+
+      - name: Setup docs workspace ⚙️
+        uses: ./.github/actions/setup-docspace
```

actionlint (1.7.7)

description is required in metadata of "Setup documentation workspace ⚙️" action at "/home/jailuser/git/.github/actions/setup-docspace/action.yml"

(action)